liquibase-service/src/main/resources/db/odse/routines/021_sp_observation_event-002.sql

```diff
@@ -0,0 +1,467 @@
+CREATE OR ALTER PROCEDURE [dbo].[sp_observation_event] @obs_id_list nvarchar(max)
```

Comment:
It would be good to take up an enhancement ticket to add in data length constraints to the sproc.